aws/table_aws_config_rule.go

```diff
@@ -186,6 +193,31 @@ func getConfigRuleTags(ctx context.Context, d *plugin.QueryData, h *plugin.Hydra
 	return op, nil
 }
 
+func getComplianceByConfigRules(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getComplianceByConfigRules")
+
+	// Create Session
+	svc, err := ConfigService(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("getComplianceByConfigRules", "connection", err)
+		return nil, err
+	}
+	ruleName := h.Item.(*configservice.ConfigRule).ConfigRuleName
+
+	// Build params
+	params := &configservice.DescribeComplianceByConfigRuleInput{
+		ConfigRuleNames: []*string{ruleName},
```

Comment:
```suggestion
		ConfigRuleNames: []*string{ruleName}
```

Reply:
It is the syntax of golang we can not remove it, If we will remove it we will get the syntax error

Reply:
Thanks !!